app/models/project.rb

```diff
@@ -57,12 +57,15 @@ class Project < ActiveRecord::Base
 
   validates_inclusion_of :private, :live, in: [true, false], message: "must be true or false"
 
+  validates :featured, uniqueness: {conditions: -> { featured } }
```

Comment:
this seems like it's overly complex, couldnt the client control how many are featured with a page_size = 1 and a sort order? Also that way we get multiple featured projects and leave the client to figure out which to display?

Reply:
Yeah, that was my thought too, but @beckyrother was very against supporting multiple featured projects. I'm happy to not provide any API support for it if you want.

Reply:
right - that is until they do ;) Up to you re leaving it and the implementation is fine.